pyradiosky/skymodel.py

```diff
@@ -55,15 +56,14 @@ class SkyModel(object):
         Reference frequencies of flux values, shape (Ncomponents,).
     spectral_type : str
         Indicates how fluxes should be calculated at each frequency.
-        Options :
-            - 'flat' : Flat spectrum.
-            - 'full' : Flux is defined by a saved value at each frequency.
-            - 'subband' : Flux is given at a set of band centers. (TODO)
-            - 'spectral_index' : Flux is given at a reference frequency. (TODO)
+        Options:
+        - 'flat' : Flat spectrum.
+        - 'full' : Flux is defined by a saved value at each frequency.
+        - 'subband' : Flux is given at a set of band centers. (TODO)
+        - 'spectral_index' : Flux is given at a reference frequency. (TODO)
```

Comment:
We should check how this renders with Sphinx.

Reply:
Yes we should... however we don't have a docs folder yet?